### ISSUE TITLE
Any Typescript Guys offer a Hand with Filter definition?

### ISSUE THREAD

**Issue by clark-stevenson** [photonstorm/phaser#843]
The Filters thing is killing me. 

See the definition:

https://github.com/photonstorm/phaser/blob/dev/build/phaser.d.ts

See these issues: 

http://www.html5gamedevs.com/topic/6591-phaser-typescript-filters/?hl=filter

http://www.html5gamedevs.com/topic/5823-inclusion-of-pixis-abstractfilter-into-phaser-build/?hl=filter

And see my Answer by Basarat which does make sense, but not in the context of the PIXI/Phaser definition.  

http://stackoverflow.com/questions/23800698/typescript-definition-when-a-class-needs-to-be-dynamic

The bottom line, is I do not know how to deal with Filters in the Definition.  The PIXI definition stuff bothers me (it is just a copy and paste with no regard for Phaser).  And filters[] is specified on the PIXI.DisplayObject property. 

I would appreciate any insight here on how to correctly implement this.

**Comment by devilb** [comment]
When I apply a blur filter like this:
`this['reel' + r].filters = [ new Phaser.Filter.BlurY(this.game, 36, '36') ];`
how can I increase the intensity? e.g. to blur = 36;
